dev/DynamicDependency/Powershell/Add-PackageDependency.ps1

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation and Contributors.
+# Licensed under the MIT License.
+
+<#
+.SYNOPSIS
+TODO
+#>
+[CmdletBinding(SupportsShouldProcess=$true)]
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$PackageDependencyId,
+
+    [int]$Rank=0,
+
+    [switch]$PrependIfRankCollision
+)
+
+Set-StrictMode -Version 3.0
+
+$ErrorActionPreference = "Stop"
+
+# Import the MSIX Dynamic Dependency module
+if (-not (Get-Module | Where-Object {$_.Name -eq 'MsixDynamicDependency'}))
+{
+    $module = Join-Path $PSScriptRoot 'MsixDynamicDependency.psm1'
+    Import-Module -Name $module -ErrorAction Stop
+}
+
+$options = [Microsoft.Windows.ApplicationModel.DynamicDependency.AddPackageDependencyOptions]::None
+if ($PrependIfRankCollision -eq $true)
+{
+    $options = $options -bor [Microsoft.Windows.ApplicationModel.DynamicDependency.AddPackageDependencyOptions]::PrependIfRankCollision
+}
+
+$packageDependencyContext = [IntPtr]0
```

Comment:
Nit: Can we say "[IntPtr] $null" here?

Reply:
Unfortunately no. Powershell doesn't accept `$null` as a valid `IntPtr` value

![image](https://github.com/microsoft/WindowsAppSDK/assets/72222/38617795-33aa-460c-b93f-c787dac89456)